content/en/tracing/compatibility_requirements/dotnet-core.md

```diff
@@ -0,0 +1,45 @@
+---
+title: .NET Core Compatibility Requirements
+kind: documentation
+description: 'Compatibility Requirements for the .NET tracer'
+further_reading:
+    - link: 'tracing/setup/dotnet-core'
+      tag: 'Documentation'
+      text: 'Instrument Your Application'
+---
+
+The .NET Datadog Trace library is open source - view the [Github repository][1] for more information.
+
+The .NET Tracer supports automatic instrumentation on .NET Core 2.1, 3.0, and 3.1. It also supports [.NET Framework][2].
```

Comment:
Dropping 3.0 since the next paragraph says it reached end of life.
```suggestion
The .NET Tracer supports automatic instrumentation on .NET Core 2.1 and 3.1. It also supports [.NET Framework][2].
```